apps/web-mzima-client/src/app/post/location-select/location-select.component.ts

```diff
@@ -13,12 +23,14 @@ import {
   MapOptions,
   marker,
   Marker,
-  MarkerClusterGroup,
   MarkerClusterGroupOptions,
   tileLayer,
 } from 'leaflet';
 import 'leaflet.markercluster';
 import { pointIcon } from '../../core/helpers/map';
+import { decimalPattern } from '../../core/helpers/regex';
+import Geocoder from 'leaflet-control-geocoder';
+import { fromEvent, filter, debounceTime, distinctUntilChanged, tap, Subscription } from 'rxjs';
 
 @Component({
```

Comment:
add **@UntilDestroy()** before @Component()

Reply:
Ok. I can see its usage inside app.component.ts like how you describe it.